src/wizard/javascript/nextjs/index.md

```diff
@@ -5,7 +5,7 @@ support_level: production
 type: framework
 ---
 
-Configure your app automatically with [Sentry wizard](https://docs.sentry.io/platforms/javascript/guides/nextjs/#configure).
+Add Sentry automatically to your app with the [Sentry wizard](https://docs.sentry.io/platforms/javascript/guides/nextjs/#install):
```

Comment:
My motivation for making this change was that the wizard isn't mentioned in the `#configure` section of the "Getting Started" docs but rather in the `#install` section above it. 

Given that the installation command is mentioned directly below, once could argue that we don't need this sentence at all. In this case, though, I'd argue to link to the wizard (#install) in the section below where we talk about what changes the wizard actually makes. Happy to make this change if reviewers agree that this would be better.

Reply:
I think this sentence makes more sense, but I will be happy to bring this topic to a discussion afterward...In any case, thanks for improving it 🙂

Reply:
Thanks for making the change! I like having this intro sentence to orient folks who haven't heard of sentry wizard before.

We don't seem to be consistent with whether we refer to the wizard as "the Sentry wizard" or just "Sentry wizard".

I prefer with the "the". Do you mind adding "The" in line 14 as well so we're consistent in this doc at least?

Reply:
+1 for "the"